fakelibusb_test.go

```diff
@@ -288,11 +307,12 @@ func (f *fakeLibusb) empty() bool {
 
 func newFakeLibusb() *fakeLibusb {
 	fl := &fakeLibusb{
-		fakeDevices: make(map[*libusbDevice]*fakeDevice),
-		ts:          make(map[*libusbTransfer]*fakeTransfer),
-		submitted:   make(chan *fakeTransfer, 10),
-		handles:     make(map[*libusbDevHandle]*libusbDevice),
-		claims:      make(map[*libusbDevice]map[uint8]bool),
+		fakeDevices:       make(map[*libusbDevice]*fakeDevice),
+		fakeLibUsbDevices: []*libusbDevice{},
```

Comment:
this would be necessary only if fakeLibUsbDevices was a map - a zero value of a slice (nil) works just fine as the default value.

Reply:
Right. Since that became a map now this gets resolved.